netweaver/setup/swap_space.sls

```diff
@@ -0,0 +1,37 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if host == node.host %}
+
+{% set swap_file = '/var/lib/swap/swapfile' %}
```

Comment:
Should we add this as a variable in the pillar file? And put this value as default.
Just wondering if this makes sense or not.
Otherwise this hardcoded value looks good